azure-pipelines.yml

```diff
@@ -0,0 +1,62 @@
+variables:
+  # `mainJob` variable points to the matrix case that will be the main case
+  # it will publish artifacts to the Azure Blob
+  mainJob: 'Mojave-Xcode-10.2.1'
```

Comment:
Do we expect any publishing from this service?
If we expect, that looks good, we just need to fix job name. If not - let's remove it